index.html

```diff
@@ -81,146 +81,263 @@
         maxTocLevel: 4,
         inlineCSS: true
       };
-
-      var includeTerms = [ "identity profile", "entity", "entity credential", "subject", "claim" ];
     </script>
   </head>
   <body>
     <section id='abstract'>
       <p>
-A self-sovereign architecture for verifiable claims is one where the
-holder of a verifiable claim is in complete control of their
-identifier, where their verifiable claims are stored, and how they are
-used. There is currently no widely used self-sovereign,
-privacy-enhancing standard for expressing and transacting verifiable
-claims (aka: credentials, attestations) via the Web.
+We typically use claims made by others about us to gain access to
+various online services. For example, we use
+email addresses to identify ourselves to online services,
+driver's licenses to prove that we are capable of operating a
+motor vehicle, university degrees to prove we are well-trained
+and knowledgeable, and government-issued passports to travel
+between countries and access financial services. It is the
+goal of this specification to provide a standard way to
+express these sorts of claims on the Web in a way that is
+automatically verifiable.
```

Comment:
`automatically verifiable` ... I don't think it's clear what's being verified. We need language that is more like:

"express these sorts of claims on the Web such that their authorship can be cryptographically verified."